mozregression/main.py

```diff
@@ -26,6 +29,30 @@
 from mozregression.launchers import REGISTRY as APP_REGISTRY
 from mozregression.test_runner import ManualTestRunner, CommandTestRunner
 
+DEFAULT_CONF_FNAME = os.path.expanduser(os.path.join("~",
+                                                     ".mozregression.cfg"))
+
+
+def get_defaults(conf_name):
+    """
+    Get custom defaults from configuration file in argument
+    """
+    defaults = {}
+
+    if os.path.isfile(conf_name):
+        try:
+            config = SafeConfigParser()
+            config.read([conf_name])
+            defaults = dict(config.items("Defaults"))
+            print("%s loaded" % conf_name)
+        except (SystemError, Error) as err:
+            print("Error while parsing %s =>  no custom default values\n%s"
+                  % (conf_name, str(err)))
+    else:
+        print("%s not found => no custom default values" % conf_name)
```

Comment:
I don't think this message is useful - don't print anything when no config is read.